common/common.scss

```diff
@@ -94,9 +94,6 @@ html:not(.discourse-gifs-with-img)
     flex-flow: row-reverse;
     img {
       height: 42px;
```

Comment:
```suggestion
      height: 30px;
```

To make it exactly half the hight of the file.

Reply:
Oops I misunderstood your previous comment, gotcha :+1: